### ISSUE TITLE
DEV-33: configure ESLint for the project

### ISSUE THREAD

**Issue by KovDimaY** [KovDimaY/Trip-Reviewer#57]
As the project has 100% coverage, CI from Travis and coverage report integrated to CI, I think one more step towards perfectness is to configure ESLint and fix all the project to do not have any warning/errors. I think for the beginning it would be good to use just the configuration from Airbnb and maybe to change some minor rules that do not fit to the project. But in the end, the idea is to integrate ESLint to the current CI and to do not let merge PRs if there present errors/warnings.

Resources with some info:
1. [How to run ESLint in you travis CI](https://gist.github.com/ronapelbaum/ef3c3459f01b002d0d719ca205401174)

2. [Separate Linter results from Test results](https://github.com/quinkennedy/travis-github-lint-status)

To work on this issue, please use a branch **DEV-33** created from **devel**. When the work is finished - create a PR to the **devel** branch. At the end of each development circles, code from the **devel** branch (if everything is ok) goes to the **master** branch and deploys to production.

**IMPORTANT!** Before creating a PR be sure that all the tests are passing and the coverage is 100%.

**Comment by KovDimaY** [comment]
Lint is configured for the project and all the errors are fixed. Also, it is added as an additional check to the Travis CI:
![image](https://user-images.githubusercontent.com/26466644/45933863-3b41cb00-bf95-11e8-8b81-a27d933824b8.png)

It is time to create a PR and very carefully review it!!! :D

**Comment by KovDimaY** [comment]
The issue is resolved. The PR was very huge but I tried my best to review it carefully and it seems like everything still works fine. If not, then during solving next issues I hope I will catch small things and fix them easily. I do not think something very serious can happen as 100% of the code is covered and tests are passing. Also, I checked manually all the basic actions and they also work fine, so I suppose the issue can be closed...
